go/runtime/materialize.go

```diff
@@ -276,13 +276,16 @@ func drainBinding(
 	// Drain the combiner into materialization Store requests.
 	if err := combiner.Drain(func(full bool, docRaw json.RawMessage, packedKey, packedValues []byte) error {
 		// Inlined use of string(packedKey) clues compiler escape analysis to avoid allocation.
-		if _, ok := flighted[string(packedKey)]; !ok {
+		if flightedDoc, ok := flighted[string(packedKey)]; !ok {
```

Comment:
nit: flightedDoc will _always_ be "" per Go spec. Don't think we need to assign or log it.